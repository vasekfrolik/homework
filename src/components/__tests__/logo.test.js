import React from "react";
import { shallow } from "enzyme";

import Logo from "./../logo";
import svgLogo from "./../../assets/iBillboardLogo.svg";

describe("render Logo", () => {
  const logo = shallow(<Logo />);
  it("matches snapshot", () => {
    expect(logo).toMatchSnapshot();
  });

  it("renders an image with logo as a source", () => {
    expect(logo.find("img").prop("src")).toEqual(svgLogo);
  });

  it("renders a div with img", () => {
    expect(logo.contains(<img src={svgLogo} alt="Logo" />)).toEqual(true);
  });

  it("has alt set to 'Logo'", () => {
    expect(logo.find("img").prop("alt")).toEqual("Logo");
  });
});
