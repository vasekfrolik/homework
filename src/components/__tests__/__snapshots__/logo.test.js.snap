// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Logo matches snapshot 1`] = `ShallowWrapper {}`;

exports[`render logo matches snapshot 1`] = `ShallowWrapper {}`;
